lib/deqm_test_kit/measure_availability.rb

```diff
@@ -10,20 +10,42 @@ class MeasureAvailability < Inferno::TestGroup
     fhir_client do
       url :url
     end
-
-    MEASURES = ['EXM130|7.3.000', 'EXM125|7.3.000'].freeze
-
+    # rubocop:disable Metrics/BlockLength
     test do
       title 'Measure can be found'
       id 'measure-availability-01'
       description 'Selected measure with matching id is available on the server and a valid json object'
       makes_request :measure_search
+      input :selected_measure_id, type: 'radio', optional: false,
```

Comment:
can we use the same JSON.parse strategy here instead of listing out all the measure info?